PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
```

Comment:
Yes. Once we have whitelisted the hostname any we can extract the bucket names from host-style requests.